TEMPLATE.md

```diff
@@ -0,0 +1,652 @@
+## Shortcuts from raulfraile, applied on Phpstorm by Pouzor ##
+
+All shortcuts start with the `sf` prefix and are both short and intuitive:
+
+### Controller ###
+
+`sfcontroller`
+
+``` php
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    public function indexAction()
+    {
+        return $this->render('.html.twig');
+    }
+}
+```
+
+`sfcontrollera`
+
+``` php
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    /**
+     * @Route("/", name="")
+     * @Template()
+     */
+    public function indexAction()
+    {
+
+    }
+}
+```
+
+`sfaction`
+
+``` php
+public function indexAction()
+{
+    return $this->render('.html.twig');
+}
+```
+
+`sfactiona`
+
+``` php
+/**
+ * @Route("/", name="")
+ * @Template()
+ */
+public function indexAction()
+{
+
+}
+```
+
+`sfem`
+
+``` php
+$em = $this->getDoctrine()->getManager();
+```
+
+`sfrepo`
+
+``` php
+$em->getRepository('Bundle:Repository');
+```
+
+`sfforward`
+
+``` php
+$this->forward('Bundle:Controller:action', array(), array());
+```
+
+`sfredirect`
+
+``` php
+$this->redirect($this->generateUrl('route', array()));
+```
+
+`sfrender`
+
+``` php
+$this->render('Bundle:Folder:template.html.twig', array());
+```
+
+`sfgetsession`
+
+``` php
+$this->getRequest()->getSession();
+```
+
+`sfsetflash`
+
+``` php
+$this->get('session')->getFlashBag()->add('type', 'message');
+```
```

Comment:
For now, I'm only going to support the newest versions. That means some things won't work for people, but as this is for shortcuts, I'd like people to see the newest/best way of things.